src/core/agent/services/identifierService.test.ts

```diff
@@ -1597,4 +1604,25 @@ describe("Identifier service of agent", () => {
       }
     );
   });
+
+  test("getIdentifier should throw an error when KERIA is offline ", async () => {
+    signifyApi.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(false);
+    await expect(identifierService.getIdentifier("id")).rejects.toThrowError(
+      IdentifierService.KERIA_IS_DOWN
+    );
+  });
+
+  test("syncKeriaIdentifiers should throw an error when KERIA is offline ", async () => {
+    signifyApi.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(false);
+    await expect(identifierService.syncKeriaIdentifiers()).rejects.toThrowError(
+      IdentifierService.KERIA_IS_DOWN
+    );
+  });
+
+  test("getUnhandledMultisigIdentifiers should throw an error when KERIA is offline ", async () => {
+    signifyApi.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(false);
+    await expect(
+      identifierService.getUnhandledMultisigIdentifiers()
+    ).rejects.toThrowError(IdentifierService.KERIA_IS_DOWN);
```

Comment:
In my opinion we can just combine all of the offline mode tests into one big test for each service